src/components/_global/BalBreakdown/BalBreakdown.vue

```diff
@@ -9,6 +9,8 @@ type Props = {
   title?: string;
   hideItems?: boolean;
   size?: string;
+  offsetClassOverrides?: string;
+  initVertBarClassOverrides?: string;
```

Comment:
@garethfuller I wasn't sure what's the best practice in Vue, but how do I pass "custom" overrides? It seems that to get close to what Pon has in his designs I need something very specific.

Reply:
I don't think overrides are the way to go here. It looks like you need to add `md` size class option to `initVertBarSizes` to cover your `h-6 -mt-6` case. And you need some prop for a left offset maybe?

Reply:
I like the direction, but still unsure of correct implementation.

Adding something like `leftOffset` would work, but I need two more things:
1. Having "mt-4" as an `offsetClass`.
2. Having "h-6 -mt-6" as an `initVertBarSizes`.

It seems that one is "xl" and the other is indeed "md", but you can't really mix/match unless we have something like a sizing object where you can pick the sizes for each?

Something like

```
size: {
  vertBar: 'md',
  offset: 'xl',
  ...
}
```